src/routes/extras/containers/ExtrasContainer.js

```diff
@@ -78,7 +79,13 @@ class ExtrasContainer extends React.Component<ExtrasPropsType, ExtrasStateType>
     setTimeout(() => this.setState({ timeoutExpired: true }), LOADING_TIMEOUT)
 
     try {
-      const payload: Payload<Array<ExtraModel>> = await (createExtrasEndpoint(baseUrl).request({ city, language }))
+      const appSettings = new AppSettings()
+      const apiUrlOverride = await appSettings.loadApiUrlOverride()
+
+      const payload: Payload<Array<ExtraModel>> = await (createExtrasEndpoint(apiUrlOverride || baseUrl).request({
```

Comment:
Is there already a test for ExtrasContainer? If not then this is fine.

Reply:
There is no test yet.